FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -52,17 +53,27 @@ public void ExecuteSelectTest(string commandText)
             executeTest(SYSTEM_CONNECTION_STRING, commandText, null, true);
         }
 
-        [Test, Timeout(600000)]
+        [Test, Timeout(1200000)]
         [Category("slow")]
         public void ExecuteLongTest()
         {
+            var watch = System.Diagnostics.Stopwatch.StartNew();
             using var conn = new FireboltConnection(SYSTEM_CONNECTION_STRING);
             conn.Open();
             DbCommand command = conn.CreateCommand();
             command.CommandTimeout = 0; // make command timeout unlimited
-            command.CommandText = "SELECT checksum(*) FROM generate_series(1, 200000000000)";
+            var max = UserName == null ? 400000000000 : 250000000000;
```

Comment:
Let's add a comment that this is a v1/v2 check. I was so confused for a bit why do we check for the UserName.